src/Lynx/Form.purs

```diff
@@ -527,40 +736,58 @@ asyncFromTypeahead typeahead x = case toString typeahead.uri of
 mvpPage :: Page Expr
 mvpPage =
   { name: "New Campaign Request"
-  , contents:
+  , tabs:
     Data.NonEmpty.NonEmpty
       { name: "Details"
       , link: "details"
-      , contents:
-        Data.NonEmpty.singleton
-          { name: "Campaign"
-          , contents:
-            Data.NonEmpty.NonEmpty
-              mvpName
-              [ mvpTargetableInterest
-              , mvpFacebookTwitterPage
-              , mvpObjective
-              , mvpMediaBudget
-              , mvpStart
-              , mvpEnd
-              ]
-          }
+      , sections: Data.NonEmpty.singleton mvpDetailsSection
       }
       [ { name: "Creative"
         , link: "creative"
-        , contents:
-          Data.NonEmpty.singleton
-          { name: "Creative"
-          , contents:
-            Data.NonEmpty.NonEmpty
-            mvpSocialAccount
-            [
-            ]
-          }
+        , sections: Data.NonEmpty.singleton mvpCreativeSequence
         }
       ]
   }
 
+mvpDetailsSection :: TabSections Expr
+mvpDetailsSection =
+  TabSection
+    { name: "Campaign"
+    , fields:
+      Data.NonEmpty.NonEmpty
+      mvpName
+      [ mvpTargetableInterest
+      , mvpFacebookTwitterPage
+      , mvpObjective
+      , mvpMediaBudget
+      , mvpStart
+      , mvpEnd
+      ]
+    }
+
+
+mvpCreativeSequence :: TabSections Expr
+mvpCreativeSequence =
+  TabSequence
+    { name: "Creative"
+    , key: "creative"
+    , template: { name: "Social Creative"
+                , fields:
+                    Data.NonEmpty.singleton $
+                      TemplateText { default: Nothing
+                                   , required: val_ (boolean_ true)
+                                   , placeholder: val_ (string_ "I don't know what a creative is")
+                                   , maxLength: Nothing
+                                   , minLength: Nothing
+                                   }
+                }
+    , values:
+        UserInput [ { name: "Social Creative"
+                    , fields: Data.NonEmpty.singleton mvpSocialAccount
+                    }
+                  ]
+    }
```

Comment:
Can we fix this indentation so we're not using ad-doc spacing? See surrounding vals, e.g. `mvpDetailsSection`